File: PRAK604-2310817210026-MUHAMMADPUTRAAZKYALFANI.py
def eve(code, message):
    result = ''
    star = 0
    hashtag = 0

    for i in range(len(code)):
        if code[i] == ' ' and message[i] == ' ':
            result += ' '
        elif code[i] == message[i]:
            result += '*'
            star += 1
        else:
            result += '#'
            hashtag += 1

    return result, star, hashtag


if __name__ == "__main__":
    code = input().rstrip('\n')
    message = input().rstrip('\n')

    if len(code) != len(message):
        print("\nPanjang kalimat berbeda, pesan palsu")
        exit()

    result, star, hashtag = eve(code, message)

    print("\n")
    print(result)
    print("* = {}".format(star))
    print("# = {}".format(hashtag))

    if star >= hashtag:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")

File: PRAK604-2310817210026-MUHAMMADPUTRAAZKYALFANI.c
#include <stdio.h>
#include <string.h>

void eve(char code[], char message[]) {
    for (int i = 0; i < strlen(code); i++) {
        if (code[i] == ' ' && message[i] == ' ') {
            message[i] = ' ';
        } else if (code[i] == message[i]) {
            message[i] = '*';
        } else {
            message[i] = '#';
        }
    }
}

int main() {
    char code[200], message[200];

    gets(code);
    code[strcspn(code, "\n")] = '\0';

    gets(message);
    message[strcspn(message, "\n")] = '\0';

    if(strlen(code) != strlen(message)) {
        printf("\nPanjang kalimat berbeda, pesan palsu \n");
        return 0;
    }

    eve(code, message);

    int star = 0, hashtag = 0;

    for(int i = 0; i < strlen(code); i++) {
        if(message[i] == '*') {
            star++;
        } else if(message[i] == '#') {
            hashtag++;
        }
    }

    printf("\n");
    printf("%s\n",message);
    printf("* = %d\n", star);
    printf("# = %d\n", hashtag);

    if (star >= hashtag) {
        printf("Pesan Asli\n");
    } else {
        printf("Pesan Palsu\n");
    }

    return 0;
}